docs/topics/installing-vs-code-extension.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/vsc-extension-guide/master.adoc
+
+
+[id="installing-vs-code-extension_{context}"]
+= Installing the {ProductShortName} extension on {VscodePluginName}
+
+You can install the {ProductShortName} extension on {VscodePluginName} on Linux, Windows or iOS.
```

Comment:
Serial comma is missing.

iOS? That is Apple mobile OS and I doubt many people are installing the VS Code on their iPhone. macOS.